scripts/requirements.txt

```diff
@@ -1,3 +1,12 @@
 GitPython==3.1.27
 pyyaml==5.4
-requests==2.27.1
\ No newline at end of file
+requests==2.27.1
+tqdm==4.63.0
+azure-ai-ml
+azure-identity
+transformers[torch]
+wget
+pandas
+mlflow
+https://scorestorageforgeneric.blob.core.windows.net/libs/azureml_metrics-0.1.0.72521629-py3-none-any.whl

```

Comment:
This repo is public facing, are we ok sharing these? who's going to update these?

Reply:
Shipra's team owns these wheel files. They are planning on pushing these to pypi in the near future. Till then we will manually update these in the requirements.txt as and when we receive updates from Shipra's team

Reply:
Could we please split this PR into two? 

1. Supporting Model publishing
2. MLFlow HF flavor?

@novaturient95